todo/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './App.css';
+import { connect } from 'react-redux';
+import { addTodo } from './actions';
+import TodoList from './Components/TodoList';
+
```

Comment:
the app throws an error. you, propbably, mean  **c**omponents not **C**omponents